WcaOnRails/app/models/competition.rb

```diff
@@ -261,4 +285,25 @@ def end_date
       errors.add(:eventSpecs, "invalid event ids: #{invalid_events.map(&:id).join(',')}")
     end
   end
+
+  def competitions_nearby
```

Comment:
Would the name `nearby_competitions` make more sense? I like having the noun last here.